tokio-quiche/src/http3/driver/streams.rs

```diff
@@ -52,6 +53,8 @@ pub(crate) struct StreamCtx {
     pub(crate) audit_stats: Arc<H3AuditStats>,
     /// Indicates the stream sent initial headers.
     pub(crate) initial_headers_sent: bool,
+    /// First time that a HEADERS frame was not fully flushed.
+    pub(crate) first_full_flush_fail_time: Option<Instant>,
```

Comment:
Nit: Maybe `last_headers_flush_fail_time`? Since there could be more than 1 HEADERS frames sent "first" seems confusing here, and also to make it clearer that it's specifically for HEADERS.